{"name": "__main__", "customMessage": "Username already exists", "args": "()", "logLevel": "WARNING", "levelno": "30", "pathname": "d:\\dev\\code\\user_auth_app\\user_auth_app\\app.py", "filename": "app.py", "module": "app", "exceptionStackTrace": "None", "exc_text": "None", "stack_info": "None", "lineno": "47", "funcName": "register", "created": "1720354228.6190476", "msecs": "619.0", "relativeCreated": "14013.282060623169", "thread": "14228", "threadName": "Thread-2 (process_request_thread)", "processName": "MainProcess", "process": "2136", "message": "Username already exists"}
{"name": "__main__", "customMessage": "Username already exists", "args": "()", "logLevel": "WARNING", "levelno": "30", "pathname": "d:\\dev\\code\\user_auth_app\\user_auth_app\\app.py", "filename": "app.py", "module": "app", "exceptionStackTrace": "None", "exc_text": "None", "stack_info": "None", "lineno": "47", "funcName": "register", "created": "1720354280.707971", "msecs": "707.0", "relativeCreated": "66102.20551490784", "thread": "13740", "threadName": "Thread-3 (process_request_thread)", "processName": "MainProcess", "process": "2136", "message": "Username already exists"}
{"name": "__main__", "customMessage": "Username already exists", "args": "()", "logLevel": "WARNING", "levelno": "30", "pathname": "d:\\dev\\code\\user_auth_app\\user_auth_app\\app.py", "filename": "app.py", "module": "app", "exceptionStackTrace": "None", "exc_text": "None", "stack_info": "None", "lineno": "47", "funcName": "register", "created": "1720361519.340336", "msecs": "340.0", "relativeCreated": "7304734.570503235", "thread": "7984", "threadName": "Thread-4 (process_request_thread)", "processName": "MainProcess", "process": "2136", "message": "Username already exists"}
{"name": "__main__", "customMessage": "Username already exists", "args": "()", "logLevel": "WARNING", "levelno": "30", "pathname": "d:\\dev\\code\\user_auth_app\\user_auth_app\\app.py", "filename": "app.py", "module": "app", "exceptionStackTrace": "None", "exc_text": "None", "stack_info": "None", "lineno": "48", "funcName": "register", "created": "1720361649.0949442", "msecs": "94.0", "relativeCreated": "8183.431386947632", "thread": "18084", "threadName": "Thread-2 (process_request_thread)", "processName": "MainProcess", "process": "1800", "message": "Username already exists"}
{"name": "__main__", "customMessage": "Username already exists", "args": "()", "logLevel": "WARNING", "levelno": "30", "pathname": "d:\\dev\\code\\user_auth_app\\user_auth_app\\app.py", "filename": "app.py", "module": "app", "exceptionStackTrace": "None", "exc_text": "None", "stack_info": "None", "lineno": "48", "funcName": "register", "created": "1720361668.6586154", "msecs": "658.0", "relativeCreated": "27747.10249900818", "thread": "15608", "threadName": "Thread-3 (process_request_thread)", "processName": "MainProcess", "process": "1800", "message": "Username already exists"}
{"name": "__main__", "customMessage": "Username already exists", "args": "()", "logLevel": "WARNING", "levelno": "30", "pathname": "d:\\dev\\code\\user_auth_app\\user_auth_app\\app.py", "filename": "app.py", "module": "app", "exceptionStackTrace": "None", "exc_text": "None", "stack_info": "None", "lineno": "48", "funcName": "register", "created": "1720361725.4443054", "msecs": "444.0", "relativeCreated": "84532.79256820679", "thread": "17300", "threadName": "Thread-4 (process_request_thread)", "processName": "MainProcess", "process": "1800", "message": "Username already exists"}
{"name": "__main__", "customMessage": "Username already exists", "args": "()", "logLevel": "WARNING", "levelno": "30", "pathname": "d:\\dev\\code\\user_auth_app\\user_auth_app\\app.py", "filename": "app.py", "module": "app", "exceptionStackTrace": "None", "exc_text": "None", "stack_info": "None", "lineno": "48", "funcName": "register", "created": "1720361740.0233881", "msecs": "23.0", "relativeCreated": "99111.87529563904", "thread": "16968", "threadName": "Thread-5 (process_request_thread)", "processName": "MainProcess", "process": "1800", "message": "Username already exists"}
{"name": "__main__", "customMessage": "Username already exists", "args": "()", "logLevel": "WARNING", "levelno": "30", "pathname": "d:\\dev\\code\\user_auth_app\\user_auth_app\\app.py", "filename": "app.py", "module": "app", "exceptionStackTrace": "None", "exc_text": "None", "stack_info": "None", "lineno": "48", "funcName": "register", "created": "1720361954.365595", "msecs": "365.0", "relativeCreated": "5348.263502120972", "thread": "4996", "threadName": "Thread-2 (process_request_thread)", "processName": "MainProcess", "process": "10120", "message": "Username already exists"}
{"name": "__main__", "customMessage": "Username already exists", "args": "()", "logLevel": "WARNING", "levelno": "30", "pathname": "d:\\dev\\code\\user_auth_app\\user_auth_app\\app.py", "filename": "app.py", "module": "app", "exceptionStackTrace": "None", "exc_text": "None", "stack_info": "None", "lineno": "48", "funcName": "register", "created": "1720362026.006793", "msecs": "6.0", "relativeCreated": "6531.151533126831", "thread": "15600", "threadName": "Thread-2 (process_request_thread)", "processName": "MainProcess", "process": "10940", "message": "Username already exists"}
{"name": "__main__", "customMessage": "Username already exists", "args": "()", "logLevel": "WARNING", "levelno": "30", "pathname": "d:\\dev\\code\\user_auth_app\\user_auth_app\\app.py", "filename": "app.py", "module": "app", "exceptionStackTrace": "None", "exc_text": "None", "stack_info": "None", "lineno": "48", "funcName": "register", "created": "1720362045.3293893", "msecs": "329.0", "relativeCreated": "4042.789936065674", "thread": "19444", "threadName": "Thread-2 (process_request_thread)", "processName": "MainProcess", "process": "9488", "message": "Username already exists"}
{"name": "__main__", "customMessage": "Username already exists", "args": "()", "logLevel": "WARNING", "levelno": "30", "pathname": "d:\\dev\\code\\user_auth_app\\user_auth_app\\app.py", "filename": "app.py", "module": "app", "exceptionStackTrace": "None", "exc_text": "None", "stack_info": "None", "lineno": "48", "funcName": "register", "created": "1720362096.2121277", "msecs": "212.0", "relativeCreated": "1696.5279579162598", "thread": "3188", "threadName": "Thread-2 (process_request_thread)", "processName": "MainProcess", "process": "19064", "message": "Username already exists"}
{"name": "__main__", "customMessage": "Username already exists", "args": "()", "logLevel": "WARNING", "levelno": "30", "pathname": "d:\\dev\\code\\user_auth_app\\user_auth_app\\app.py", "filename": "app.py", "module": "app", "exceptionStackTrace": "None", "exc_text": "None", "stack_info": "None", "lineno": "48", "funcName": "register", "created": "1720362566.5909953", "msecs": "590.0", "relativeCreated": "163128.515958786", "thread": "19252", "threadName": "Thread-3 (process_request_thread)", "processName": "MainProcess", "process": "12012", "message": "Username already exists"}
